plugins/logs/dao.go

```diff
@@ -95,20 +94,21 @@ func (es *elasticsearch) indexRecord(ctx context.Context, rec record) {
 	}
 }
 
-func (es *elasticsearch) getRawLogs(ctx context.Context, from, size, filter string, indices ...string) ([]byte, error) {
-	offset, err := strconv.Atoi(from)
-	if err != nil {
-		return nil, fmt.Errorf(`invalid value "%v" for query param "from"`, from)
-	}
-	s, err := strconv.Atoi(size)
-	if err != nil {
-		return nil, fmt.Errorf(`invalid value "%v" for query param "size"`, size)
-	}
+type logsConfig struct {
```

Comment:
we can rename this to `logsFilter` which will more appropriate and easy to come back in future.